File: examples/swift/constructs/scope_if.swift
// Ported from examples/lumen/constructs/scope_if.lm by scripts/port_examples.py; edit the Lumen original, not this file.
var x = 10
if true {
    x = 20
    print(x)
} else {
    x = 30
    print(x)
}
print(x)

File: examples/c/constructs/scope_if.c
// Ported from examples/lumen/constructs/scope_if.lm by scripts/port_examples.py; edit the Lumen original, not this file.
int main(void) {
    long x = 10;
    if (true) {
        x = 20;
        printf("%ld\n", x);
    } else {
        x = 30;
        printf("%ld\n", x);
    }
    printf("%ld\n", x);
    return 0;
}
